Kickstarter-iOS/Views/Controllers/PledgeViewController.swift

```diff
@@ -379,6 +379,23 @@ final class PledgeViewController: UIViewController, MessageBannerViewControllerP
   @objc private func dismissKeyboard() {
     self.view.endEditing(true)
   }
+
+  private func beginSCAFlow(withClientSecret secret: String) {
+    STPPaymentHandler.shared().confirmSetupIntent(
+      withParams: .init(clientSecret: secret),
+      authenticationContext: self
+    ) { [weak self] status, _, error in
```

Comment:
Just curious what's this middle parameter here that we're ignoring?

Reply:
That's the `SetupIntent` object. I don't believe we need it for anything.